src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -46,110 +59,80 @@ const propTypes = {
-            // If this is an attachment placeholder, return the placeholder component
-            if (props.isAttachment && props.loading) {
-                return Str.isImage(props.attachmentInfo.name) ? (
-                    <RenderHTML html={`<comment><img src="${props.attachmentInfo.source}" data-expensify-preview-modal-disabled="true"/></comment>`} />
-                ) : (
-                    <View style={[styles.chatItemAttachmentPlaceholder]}>
-                        <ActivityIndicator
-                            size="large"
-                            color={themeColors.textSupporting}
-                            style={[styles.flex1]}
-                        />
-                    </View>
-                );
-            }
-            const {html, text} = props.fragment;
+    const fragment = props.fragment;
 
+    switch (fragment.type) {
+        case 'COMMENT': {
             // Threaded messages display "[Deleted message]" instead of being hidden altogether.
             // While offline we display the previous message with a strikethrough style. Once online we want to
             // immediately display "[Deleted message]" while the delete action is pending.
 
-            if ((!props.network.isOffline && props.hasCommentThread && props.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) || props.fragment.isDeletedParentAction) {
-                return <RenderHTML html={`<comment>${props.translate('parentReportAction.deletedMessage')}</comment>`} />;
-            }
-
-            // If the only difference between fragment.text and fragment.html is <br /> tags
-            // we render it as text, not as html.
-            // This is done to render emojis with line breaks between them as text.
-            const differByLineBreaksOnly = Str.replaceAll(html, '<br />', '\n') === text;
+            const isPendingDelete = props.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE;
+            const isPendingOnlineDelete = !props.network.isOffline && isPendingDelete;
 
-            // Only render HTML if we have html in the fragment
-            if (!differByLineBreaksOnly) {
-                const isPendingDelete = props.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE && props.network.isOffline;
-                const editedTag = props.fragment.isEdited ? `<edited ${isPendingDelete ? 'deleted' : ''}></edited>` : '';
-                const htmlContent = applyStrikethrough(html + editedTag, isPendingDelete);
+            if ((props.hasCommentThread && isPendingOnlineDelete) || fragment.isDeletedParentAction) {
+                return <RenderCommentHTML
+                    source={props.source}
+                    html={props.translate('parentReportAction.deletedMessage')}
+                />;
+            }
 
-                return <RenderHTML html={props.source === 'email' ? `<email-comment>${htmlContent}</email-comment>` : `<comment>${htmlContent}</comment>`} />;
```

Comment:
This is the most typical point where the attachments were rendered, right in the middle of the logic focused on adding strikethrough to the text

Reply:
Actually, I might've lost the `editedTag` for attachments... it might be used

Reply:
Can you edit an attachment though?...

Reply:
No, you can't 👍